ui/raidboss/popup-text.ts

```diff
@@ -1194,6 +1212,30 @@ export class PopupText {
     this._addTextFor('info', triggerHelper);
   }
 
+  _onTriggerInternalRumble(triggerHelper: TriggerHelper): void {
+    if (!this.options.RumbleEnabled)
+      return;
+
+    // getGamepads returns a "GamePadList" which isn't iterable.
+    [...navigator.getGamepads()].forEach((gp) => {
+      if (!gp)
```

Comment:
From MDN documentation:

```
Elements in the array may be null if a gamepad disconnects during a session, so that the remaining gamepads retain the same index. 
```